WindowsAzureMessaging/WindowsAzureMessaging/Config/Global.xcconfig

```diff
@@ -36,10 +36,6 @@ GCC_NO_COMMON_BLOCKS = YES
 // :Mark: Preprocessing
 ENABLE_STRICT_OBJC_MSGSEND = YES
 
-// This flag adds -fembed-bitcode-marker in the release configuration
-// but we require the full bitcode flag -fembed-bitcode in this case.
-ENABLE_BITCODE = NO;
```

Comment:
Just removing this line is also incorrect. See https://github.com/microsoft/appcenter-sdk-apple/pull/2094 as reference, or just copy config from `develop`